x-pack/plugins/observability_solution/profiling/public/components/frame_information_window/apm_transactions.tsx

```diff
@@ -78,6 +80,21 @@ const findServicesAndTransactions = (
   };
 };
 
+function EstimatedLabel({ label }: { label: string }) {
+  return (
+    <EuiToolTip
+      content={i18n.translate('xpack.profiling.functionsView.samplesColumnLabel.hint', {
+        defaultMessage: 'Estimated values',
+      })}
+    >
+      <>
+        {label}{' '}
+        <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
```

Comment:
nit: I saw that the alignment we have in other tables is setting the icon next to the content so I think you can remove the class ( Hosts view example: 
<img width="569" alt="image" src="https://github.com/elastic/kibana/assets/14139027/5c1582c1-c50b-47ae-8c00-bafc94443bb2"> )
```suggestion
        <EuiIcon size="s" color="subdued" type="questionInCircle" />
```